src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseDeprecatedSecurityProtocols.cs

```diff
@@ -83,6 +85,30 @@ public override void Initialize(AnalysisContext context)
                                         constantValue));
                             }
                         }, OperationKind.FieldReference);
+                    compilationStartAnalysisContext.RegisterOperationAction(
+                        (OperationAnalysisContext operationAnalysisContext) =>
+                        {
+                            var assignmentOperation = (IAssignmentOperation)operationAnalysisContext.Operation;
+                            if (!securityProtocolTypeTypeSymbol.Equals(assignmentOperation.Target.Type))
+                            {
+                                return;
+                            }
+
+                            if (assignmentOperation.Value.HasAnyOperationDescendant(
```

Comment:
Instead of looking for any descendant, wouldn't looking at just the `IAssignmentOperation.Value` suffice? Or does that not have a constant value?

Reply:
Not necessarily a constant.  I want to handle things like this (will add another unit test):
```csharp
ServicePointManager.SecurityProtocol = ServicePointManager.SecurityProtocol | (SecurityProtocolType)192;
```